db/schema.rb

```diff
@@ -247,33 +247,6 @@
     t.index ["uuid"], name: "index_events_on_uuid", unique: true, length: 36
   end
 
-  create_table "lhmn_events", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin", force: :cascade do |t|
```

Comment:
I am not sure I don't understand why is this tabled being removed. Could you elaborate?

Reply:
This is a temporary table generated by the `lhm` gem.